### DISCUSSION TITLE
How to replace ApolloClient in graphQLClient option

### DISCUSSION THREAD

**Discussion by sirganya** [accounts-js/accounts#1139] [Q&A]
Hi,

Is it possible to replace Apollo client with a roll you own? I use graphql-request.

```
const accountsGraphQL = new GraphQLClient({
  graphQLClient: apolloClient,
  userFieldsFragment: gql`...
```

**Comment by pradel** [comment]
It's not possible out of the box but you can achieve this by mapping the `query` and `mutate` methods we use to graphql-request. 
This is how we use them: https://github.com/accounts-js/accounts/blob/master/packages/graphql-client/src/graphql-client.ts#L231
